addons/website_livechat/static/tests/tours/website_livechat_chatbot_test_page_tour.js

```diff
@@ -45,5 +45,13 @@ registry.category("web_tour.tours").add("website_livechat_chatbot_test_page_tour
                 });
             },
         },
+        {
+            trigger: '.o-livechat-root:shadow li:contains("Other & Documentation")',
+            run: "click",
+        },
+        {
+            trigger: messagesContain("Please find documentation at"),
+            run: "click",
```

Comment:
It's a `text` step. Click does not seem to make sense? What is the intent?

Reply:
click is probably useless, I saw steps above doing useless clicks so I didn't test without putting `run: click`. I'll try it without